### PR TITLE
[RW-5576][RISK=NO]  UI: Update concept set UI - duplicate shopping cart function

### PR DESCRIPTION
**Pull request by NehaBroad**
Screen Shot 2020-09-24 at 9 54 56 AM
Screen Shot 2020-09-24 at 9 55 21 AM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/data/concept/concept-homepage.tsx [hunk 1]
```diff
@@ -553,8 +593,27 @@ export const ConceptHomepage = withCurrentWorkspace()(
...
+                            onSelectConcepts={this.selectConcepts.bind(this)}
+                            placeholderValue={this.noConceptsConstant}
+                            searchTerm={this.state.currentSearchString}
+                            selectedConcepts={selectedConceptDomainMap[activeDomainTab.domain]}
+                            reactKey={activeDomainTab.name}
+                            error={domainError}/>
+              {!this.domainLoading(activeDomainTab) && <Button style={{float: 'right', marginBottom: '2rem'}}
+                    disabled={this.activeSelectedConceptCount === 0 ||
+                    !this.state.workspacePermissions.canWrite}
+                    onClick={() => this.setState({conceptAddModalOpen: true})}>Finish</Button>}
```
Comment (dolbeew) [inline ui/src/app/pages/data/concept/concept-homepage.tsx:614]: Sorry, I missed this the first time. Not sure how closely we need to mirror CB at this point, but should this be a 'Finish & Review` button that opens the sidebar, then have a 'Save Concepts' button at the bottom of the selected concepts list?

Comment (NehaBroad) [reply ui/src/app/pages/data/concept/concept-homepage.tsx:614]: Good point

Comment (NehaBroad) [reply ui/src/app/pages/data/concept/concept-homepage.tsx]: Done

File: ui/src/app/components/help-sidebar.tsx [hunk 2]
```diff
@@ -18,27 +18,28 @@ import {ClrIcon} from 'app/components/icons';
...
 import {SidebarContent} from 'app/pages/data/cohort-review/sidebar-content.component';
+import {ConceptListPage} from 'app/pages/data/concept/concept-list';
 import {participantStore} from 'app/services/review-state.service';
 import colors, {colorWithWhiteness} from 'app/styles/colors';
 import {
   highlightSearchTerm,
   reactStyles,
   ReactWrapperBase,
-  withCurrentCohortCriteria,
+  withCurrentCohortCriteria, withCurrentConcept,
```
Comment (dolbeew) [inline ui/src/app/components/help-sidebar.tsx:28]: nit: move to new line

File: ui/src/app/components/help-sidebar.tsx [hunk 3]
```diff
@@ -18,27 +18,28 @@ import {ClrIcon} from 'app/components/icons';
...
   withCurrentWorkspace,
   withUserProfile
 } from 'app/utils';
 import {AnalyticsTracker} from 'app/utils/analytics';
 import {
-  currentCohortSearchContextStore,
+  currentCohortSearchContextStore, currentConceptStore,
```
Comment (dolbeew) [inline ui/src/app/components/help-sidebar.tsx:34]: nit: move to new line

File: ui/src/app/pages/data/concept/concept-homepage.tsx [hunk 4]
```diff
@@ -14,17 +15,43 @@ import {ConceptAddModal} from 'app/pages/data/concept/concept-add-modal';
...
+  ReactWrapperBase,
+  validateInputForMySQL,
+  withCurrentConcept,
+  withCurrentWorkspace
+} from 'app/utils';
+import {
+  currentConceptStore,
+  NavStore,
+  queryParamsStore,
+  serverConfigStore, setSidebarActiveIconStore
```
Comment (dolbeew) [inline ui/src/app/pages/data/concept/concept-homepage.tsx:30]: nit: move to new line

File: ui/src/app/pages/data/concept/concept-homepage.tsx [hunk 5]
```diff
@@ -359,6 +391,7 @@ export const ConceptHomepage = withCurrentWorkspace()(
       const {namespace, id} = this.props.workspace;
       this.setState({completedDomainSearches: [], concepts: [], countsError: false,
         domainErrors: [], countsLoading: true, searching: true});
+      currentConceptStore.next([]);
```
Comment (dolbeew) [inline ui/src/app/pages/data/concept/concept-homepage.tsx:394]: I think this should happen only if `enableConceptSetSearchV2` is `true`. When I deploy with the flag disabled, I still see the shopping cart icon in the sidebar.

Screen Shot 2020-09-28 at 2 19 44 PM

**Review by dolbeew — APPROVED** [review]

**Pull request merged**